.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           black . --check --diff
       - name: Lint
         run: |
-          pylint salutation.py test_salutation.py
+          pylint salutation.py salutation_test.py
```

Comment:
A few thoughts about this:

- As far as I can tell the most common convention in Python code is to put the main module code inside a subdirectory with the same name as the application. This way the application code is separated from any non-module files and directories [Makefiles and build outputs, which live in the top level directory, and infrastructure code, which usually lives in a sibling directory.
- Tests are usually placed in a directory called `tests` at the same level as the module.
- Once the application starts including lots of tests it is common to have more than one test file for a single production file. For example, there might be separate files for unit and acceptance tests of a file, or for model and view, or for backend and frontend.

So in this application I would expect to see something like `hello_world/salutation.py` and `tests/test_salutation.py`.

Reply:
thanks for the review @l0b0. I have checked the [official doc ](https://docs.pytest.org/en/stable/goodpractices.html#tests-as-part-of-application-code)and you are right about putting tests in a subdirectory. In that case I guess prefix / suffix doesn't matter? And since the doc uses prefix I will keep it as prefix in this repo